distribution/docs/src/main/resources/_contents/_endpoints/developing-endpoints-contents.adoc

```diff
@@ -0,0 +1,52 @@
+
+=== Developing Endpoints
+
+Complete the following procedure to create an endpoint. 
+
+. Create a Java class that implements the endpoint's business logic. Example: Creating a web service that external clients can invoke.
```

Comment:
Wiring (e.g. jaxws or jaxrs in blueprint) is left as an exercise for the reader? 😄